Content.Client/GameObjects/Components/HUD/Inventory/ClientInventoryComponent.cs

```diff
@@ -214,5 +214,20 @@ public bool TryGetSlot(Slots slot, out IEntity item)
         {
             return _slots.TryGetValue(slot, out item);
         }
+
+        public bool TryFindItemSlots(IEntity item, out Slots slots)
+        {
+            foreach (var pair in _slots)
+            {
+                if (pair.Value == item)
+                {
+                    slots = pair.Key;
+                    return true;
+                }
+            }
+
+            slots = Slots.NONE;
+            return false;
+        }
```

Comment:
Would be better to have the file marked as nullable and return a null slot instead IMO.

```suggestion
        public bool TryFindItemSlots(IEntity item, [NotNullWhen(true)] out Slots? slots)
        {
            slots = null;
        
            foreach (var (slot, entity) in _slots)
            {
                if (entity == item)
                {
                    slots = slot;
                    return true;
                }
            }

            return false;
        }
```